src/ui/map.js

```diff
@@ -1795,19 +1944,35 @@ class Map extends Camera {
     /**
      * Sets the filter for the specified style layer.
      *
+     * Filters control which features a style layer renders from its source.
+     * Any feature for which the filter expression evaluates to true will be
+     * rendered on the map. Those that are false will be hidden.
+     *
+     * Use setFilter to show a subset of your source data.
```

Comment:
We could reference/link this function, otherwise wrap it in `.

Reply:
Sounds good. JFYI, I discovered that our internal links to class members (`Map#getZoom`, etc) aren't working right now, so I've just been wrapping these in backticks. There's an issue here (and some links to other, _very old_, related issues): https://github.com/mapbox/mapbox-gl-js-docs/issues/287.